### PR TITLE
[AccessorySetupKit] Implement Xcode 16.0 beta 1-6 changes.

### PR DESCRIPTION
**Pull request by rolfbjarne**
Note: there were no changes in beta 2 or beta 3.

### CODE REVIEW

File: src/accessorysetupkit.cs [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+		void ShowPicker (ASPickerDisplayItem [] displayItems, ASAccessorySessionCompletionHandler completionHandler);
+
+		[Export ("finishAuthorization:settings:completionHandler:")]
+		void FinishAuthorization (ASAccessory accessory, ASAccessorySettings settings, ASAccessorySessionCompletionHandler completionHandler);
+
+		[Export ("removeAccessory:completionHandler:")]
+		void RemoveAccessory (ASAccessory accessory, ASAccessorySessionCompletionHandler completionHandler);
+
+		[Export ("renameAccessory:options:completionHandler:")]
+		void RenameAccessory (ASAccessory accessory, ASAccessoryRenameOptions renameOptions, ASAccessorySessionCompletionHandler completionHandler);
```
Comment (dalexsoto) [inline src/accessorysetupkit.cs:110]: Any reason for not doing these `[Async]`?

File: src/accessorysetupkit.cs [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+	[DisableDefaultCtor]
+	interface ASPickerDisplayItem {
+		[Export ("allowsRename", ArgumentSemantic.Assign)]
+		bool AllowsRename { get; set; }
+
+		[Export ("renameOptions", ArgumentSemantic.Assign)]
+		ASAccessoryRenameOptions RenameOptions { get; set; }
+
+		[Export ("name", ArgumentSemantic.Copy)]
+		NSString Name { get; set; }
```
Comment (dalexsoto) [inline src/accessorysetupkit.cs:182]: why `NSString` here but `.ctor` bellow is `string`?

**Review by dalexsoto — APPROVED** [review]
just some minor questions

**Review by mandel-macaque — APPROVED** [review]
Approving, yet I agree with @dalexsoto  comments.

File: src/accessorysetupkit.cs (hunk 1 above)
Comment (rolfbjarne) [reply src/accessorysetupkit.cs]: Fixed to be [Async].

File: src/accessorysetupkit.cs (hunk 2 above)
Comment (rolfbjarne) [reply src/accessorysetupkit.cs]: Fixed.

**Review by haritha-mohan — APPROVED** [review]

**Pull request merged**